packages/default/scss/toolbar/_layout.scss

```diff
@@ -148,11 +223,39 @@
                 margin-inline-start: calc( #{$_spacing} * -1 );
             }
 
+            &.k-toolbar-scrollable::before {
+                inset-inline-start: $_spacing;
+            }
+
             > * > label {
                 margin-inline-end: $_spacing;
             }
+
+            .k-toolbar-items {
+                gap: $_spacing;
+            }
+
+        }
+
+        .k-toolbar-items-list-#{$size} {
+            padding-block: $_padding-y;
+            padding-inline: $_padding-x;
+            gap: $_spacing;
         }
     }
+
+    .k-toolbar-sm .k-toolbar-button-separator {
```

Comment:
I think these styles can be improved. You could include the calculated size of the button-separator in the `$kendo-toolba-sizes` instead of having separate rules for each button-separator in each size. 

> NB: Also applies to the Fluent theme